src/modules/uxrce_dds_client/dds_topics.h.em

```diff
@@ -75,7 +75,7 @@ void SendTopicsSubs::init() {
 	for (unsigned idx = 0; idx < sizeof(send_subscriptions)/sizeof(send_subscriptions[0]); ++idx) {
 		fds[idx].fd = orb_subscribe(send_subscriptions[idx].orb_meta);
 		fds[idx].events = POLLIN;
-		orb_set_interval(fds[idx].fd, send_subscriptions[idx].polling_interval);
+		orb_set_interval(fds[idx].fd, send_subscriptions[idx].max_rate_hz ? 1000000 / send_subscriptions[idx].max_rate_hz : 0);
```

Comment:
This means `0 Hz` has the special meaning of `unlimited`.

Reply:
This may not even matter, but isn't that the opposite of what it should mean? A setting of 0 Hz meaning fully off is what would follow continuously.

Imagine someone updating the topics.yaml (or hopefully a runtime equivalent soon) and they want to temporarily disable a topic, so they see the max_rate_hz and temporarily change it to 0.

Reply:
Hm, this makes sense. But also, "unlimited" is a nice feature to reduce the delay to a maximum.

What about using `> 1000` to be explicitly unlimited? It would already be implicitly unlimited from 1001 onwards as `1000/1001 = 0` as it is an int and base 1000 in PX4. (base 1000000 in NuttX's uORB)